CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 3.31.0 (UNRELEASED)
+
+### Adds
+
+* Adds the widgets placeholder feature.  
+Non-contextual widgets now take the `placeholder` option. Those having this option set to `true` that are added to an area will be inserted directly on the page with the `placeholder` attribute from their fields as rendered data, without having the initial edit modal displayed.  
+A default `.apos-placeholder` class is added to their wrapper, showing that they are rendered with their placeholder data. This class can be overridden with the `placeholderClass` widget option, in order to apply a custom style on a particular project.  
```

Comment:
The `.` is part of a selector, it is not part of the class name, so leave out the `.` here.

Reply:
Yeah that's what I thought